openff/toolkit/tests/test_molecule.py

```diff
@@ -370,6 +371,53 @@ def test_set_molecule_error(self):
         with pytest.raises(AssertionError, match="already has an associated molecule"):
             atom.molecule = mol
 
+    @pytest.fixture()
+    def water_without_charges(self):
+        return Molecule.from_smiles("O")
```

Comment:
(not blocking) Since molecules in test code tend to get reused a lot, it may be cleaner to use from_mapped_smiles here so we know what to expect at each atom index.